lua/nvim-treesitter-playground/internal.lua

```diff
@@ -264,6 +265,13 @@ function M.highlight_nodes(bufnr, nodes)
   end
 end
 
+function M.goto_node(bufnr)
+  utils.for_each_buf_window(bufnr, function(window)
+    api.nvim_set_current_win(window)
+    return
```

Comment:
/nit Is the return needed here?

Reply:
not really. if you remove it cursor would go to last window of the buffer instead of the first.

Reply:
But the return is for the callback though. The return value isn't used in `for_each_buf_window`.

Reply:
:facepalm: right!